src/Aggregate/AggregateDescription.php

```diff
@@ -80,6 +83,57 @@ public static function describe(EventEngine $eventEngine): void
                 ->storeEventsIn(EventEngineUtil::fromAggregateNameToStreamName($aggregateName))
                 ->storeStateIn(EventEngineUtil::fromAggregateNameToDocumentStoreName($aggregateName));
         }
+
+        foreach (static::projectorsAggregateMapping() as $projectorClass) {
+            $eventsForProjector = $projectorClass::getEvents();
```

Comment:
I would first check if the $projectorClass implements the Projector interface so the getEvents static method can be called.

Reply:
That is indeed cleaner